src/common/addr_parsing.c

```diff
@@ -126,26 +128,31 @@ char *resolve_addrs(const char *orig_str)
 	host, port,
 	res->ai_flags, res->ai_family, res->ai_socktype, res->ai_protocol,
 	res->ai_canonname);*/
-      if (res->ai_family == AF_INET6)
+      if (res->ai_family == AF_INET6) {
```

Comment:
I appreciate wanting to use curly braces in `if` statements but this just introduces code churn which at the very least complicates `git blame`. Please take these out.

Reply:
@batrick this comment is for all the single statement if blocks?